ddp-workspace/projects/ddp-cgc/src/app/components/footer/footer.component.html

```diff
@@ -0,0 +1,21 @@
+<footer _ngcontent-c2="" class="footer row">
+  <div _ngcontent-c2="" class="col-lg-6 col-md-4 col-sm-12 col-xs-12 footer-logo">
+      <a _ngcontent-c2="" routerLink="/home">
+          <img _ngcontent-c2="" alt="Cancer Collective Logo" src="/assets/images/logo-footer.svg">
+      </a>
+  </div>
+  <div _ngcontent-c2="" class="col-lg-3 col-md-4 col-sm-6 col-xs-12 mt10">
+      <p _ngcontent-c2="" class="footer-text">
+          <span _ngcontent-c2="" class="medium">Contact Us</span><br _ngcontent-c2="">
+          <span _ngcontent-c2="" class="light">info@cancercollective.org<br>651-229-5806</span>
+      </p>
+  </div>
+  <div _ngcontent-c2="" class="col-lg-3 col-md-4 col-sm-6 col-xs-12 mt10">
+      <p _ngcontent-c2="" class="footer-text float--right">
+          <a _ngcontent-c2="" class="footer-link" [routerLink]="Route.Home">Home</a>
+          <a _ngcontent-c2="" class="footer-link" [routerLink]="Route.AboutUs">About Us</a>
+          <a _ngcontent-c2="" class="footer-link" [routerLink]="Route.LearnMore">Learn More</a>
+          <button _ngcontent-c2="" class="footer-link"><i class="fa fa-arrow-up footer-arrow"></i> Back to top</button>
```

Comment:
Should we pull these out to `en.json` translation file? There's a few other files where I saw hardcoded text instead translation too. Can you double-check?

Reply:
I checked text without internationalisation and fixed it.